source/agent/quic/dist.json

```diff
@@ -29,7 +29,8 @@
               "../connections.js",
               "../InternalConnectionFactory.js",
```

Comment:
You can remove "../InternalConnectionFactory.js",

Reply:
Removed.